First Spanish translation update

EdgarLux opened the pull request:
First adaptation and update of Spanish translation for Ansel.

aurelienpierre commented:
Muchas gracias ! I will force-push manually.

aurelienpierre commented:
@EdgarLux it seems there are duplicated strings in the .po that make the compilation fail. Do you have a way to fix that ?

EdgarLux commented:
@aurelienpierre I guess master was updated before .po was integrated. I'm going to check it to fix it.

EdgarLux commented:
> @EdgarLux it seems there are duplicated strings in the .po that make the compilation fail. Do you have a way to fix that ?
@aurelienpierre I guess it is now fixed, ready and updated to be integrated.

aurelienpierre commented:
Ok closing that since Spanish translation is now outdated
